org.eclipse.tm4e.core/META-INF/MANIFEST.MF

```diff
@@ -9,7 +9,8 @@ Require-Bundle: org.apache.batik.css;bundle-version="1.9.1";resolution:=optional
  org.apache.batik.util;bundle-version="1.9.1";resolution:=optional,
  com.google.gson;resolution:=optional,
  org.jcodings;bundle-version="1.0.18",
- org.joni;bundle-version="2.1.11"
+ org.joni;bundle-version="2.1.11",
+ org.yaml.snakeyaml;bundle-version="1.27.0"
```

Comment:
I wonder if there are a lot of textmate grammar written in YAML? If no, perhaps we could set this bundle as optional?

Reply:
VSCode supports it so I think it would be good to support it in tm4e out-of-the-box too. Writing grammars in YAML has the advantage over JSON that it's more compact and supports aliases and comments.

Reply:
OK thanks for your feedback. With JSON you can too write  coments (jsonc)

It makes sense to have yaml parser  support.